src/token/components/TokenImage.tsx

```diff
@@ -1,7 +1,10 @@
 import { useMemo } from 'react';
 import { TokenImageReact } from '../types';
+import { getTokenImageColor } from './getTokenImageColor';
+
+export function TokenImage({ token, size = 24 }: TokenImageReact) {
+  const { image, symbol, name } = token;
 
-export function TokenImage({ src, size = 24 }: TokenImageReact) {
   const styles = useMemo(() => {
     return {
```

Comment:
Can all of this be converted in Tailwind vibes? what do you think? or is it not the right approach.

Reply:
hmmm can do

Reply:
Actually let's do that in a separate PR and discuss it separately.

To not clash with user's tailwind config, we might need to add a prefix to all the tailwind properties which can get a bit messy.

Reply:
perfect